src/etc/htmldocck.py

```diff
@@ -240,9 +240,37 @@ def concat_multi_lines(f):
         print_err(lineno, line, 'Trailing backslash at the end of the file')
 
 
+def get_known_directive_names():
+    def filter_line(line):
+        line = line.strip()
+        return line.startswith('"') and (line.endswith('",') or line.endswith('"'))
+
+    # Equivalent to `src/tools/compiletest/src/header.rs` constant of the same name.
+    with open(
```

Comment:
Oh, I briefly forgot that htmldocck isn't in Python yet (#125780) hence my nonsensical `include!` suggestion earlier.

Thanks for applying my suggestion. I guess the added runtime cost is unavoidable rn :| (reopening this for each test). I guess that means I have to prioritize #125780 again.

Reply:
Yep not great but better than duplicated code. :)